help/print-a-topic.md

```diff
@@ -0,0 +1,12 @@
+# Print a topic
+
+Topics are printable in Zulip. To save paper, printed topics exclude sidebars
+and other UI features that you otherwise see on screen. And whether you're
+viewing Zulip in the [light or dark theme](/help/dark-theme), Zulip topics will
+print with appropriate colors, or their grayscale equivalents, for paper or
+system-generated PDFs. Background colors will not be printed.
+
+Topics will print beginning with the oldest loaded message you can see on
+screen. If you need to print messages from further back in time on a long topic,
+you'll need to scroll up through the topic history until you can see the
+messages you wish to print on screen.
```

Comment:
So, what we want to explain here is that the printed view starts with the oldest message currently displayed on the screen and will include all subsequent messages in the conversation even if they aren't on the screen, correct? And then maybe if you want to print the entire conversation available to you, then you would want to navigate to the first message in the topic?

Reply:
Yes. I've tried to clarify that in the latest push.